packages/cardpay-sdk/sdk/reward-manager/base.ts

```diff
@@ -32,13 +37,20 @@ import { ZERO_ADDRESS } from '../constants';
 import { WithSymbol, RewardTokenBalance } from '@cardstack/cardpay-sdk';
 import { query } from '../utils/graphql';
 import { Signer } from 'ethers';
+import { get } from 'lodash';
 
 export interface RewardProgramInfo {
   rewardProgramId: string;
   rewardProgramAdmin: string;
   locked: boolean;
-  rule: string;
+  blob: string;
```

Comment:
Just out of curiosity, what does this blob attribute will hold?

Reply:
It is hex encoded version of the did string. For example, the blob for 0xab20c80fcc025451a3fc73bB953aaE1b9f640949 equals 0x6469643a63617264737461636b3a317236705438593943416d4a5a7066624858394d5362577763393639313538623466306334323461